clang/lib/CConv/DeclRewriter.cpp

```diff
@@ -110,8 +111,10 @@ void DeclRewriter::rewriteDecls(ASTContext &Context, ProgramInfo &Info,
   // Build sets of variables that are declared in the same statement so we can
   // rewrite things like int x, *y, **z;
   GlobalVariableGroups GVG(R.getSourceMgr());
+  FieldFinder FF(GVG);
   for (const auto &D : TUD->decls())
-    GVG.addGlobalDecl(dyn_cast<VarDecl>(D));
+    GVG.addGlobalDecl(dyn_cast<VarDecl>(D)),
+      FF.TraverseDecl(D);
```

Comment:
This is a little odd. You are comma-separating these two calls? Is this meant to be within the loop, or outside it? I don't know what will happen with the comma separation.